src/components/TabMenu.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Button, Typography, Container, AppBar, Tabs, Tab } from '@material-ui/core';
+import { TabContext, TabPanel } from '@material-ui/lab';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import Record from '../components/Record';
+import { InvoiceRecord, PaymentRecord, MeterReadingRecord } from '../utils/airtable/interface';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      marginTop: '10px',
+      border: '1px solid',
+      borderColor: theme.palette.divider,
+      borderRadius: '6px',
+    },
+    scroll: {
+      maxHeight: '380px',
+      overflow: 'auto',
+    },
+    tabs: {
+      background: 'white',
+      outline: '3px solid white',
+      border: '1px solid white',
+    },
+    tab: {
+      color: '#e5e5e5',
+      fontSize: '12px',
+      '&:focus': {
+        outline: 'none',
+      }
+    },
+    appBar: {
+      boxShadow: 'none',
+    }
+  }),
```

Comment:
you can probably fix the spacing that was removed in the Records component from before here, by adding a `tabPanel` class to assign to the TabPanel components
```
    tabPanel: {
      padding: '10px 0 0',
    },
```